src/index.js

```diff
@@ -186,6 +195,12 @@ class HCaptcha extends React.Component {
       });
     }
 
+    handleOnLoadingError (event) {
+      const { onError } = this.props;
+
+      if (onError) onError(event);
+    }
+
```

Comment:
Let's utilize the `handleError` below, we can update it like so:

```js
 handleError (event) {
      const { onError } = this.props;
      const { captchaId } = this.state;

      if (this.isReady())
           // If hCaptcha runs into error, reset captcha - hCaptcha
           hcaptcha.reset(captchaId)

      if (onError) onError(event);
    }
```

Reply:
good idea, thanks. refactored